WordPress/Resources/en-GB.lproj/Localizable.strings

```diff
@@ -9,16 +9,13 @@
 /* Message of Delete Site confirmation alert; substitution is site's host. */
 "\nTo confirm, please re-enter your site's address before deleting.\n\n" = "\nTo confirm, please re-enter your site's address before deleting.\n\n";
 
-/* No comment provided by engineer. */
-" audio file" = " audio file";
-
 /* Title for the lazy load images setting */
 "\"Lazy-load\" images" = "\"Lazy-load\" images";
 
 /* Title of a list of buttons used for sharing content to other services. These buttons appear when the user taps a `More` button. */
 "\"More\" Button" = "\"More\" Button";
 
-/* Description for the restore action. $1$@ is a placeholder for the selected date. */
+/* No comment provided by engineer. */
```

Comment:
🤔 The comment deletion happened in the source that generated the `.strings` on which this is bases, so that's out of scope for this PR, but the comment was actually helpful, it's a shame it's gone... I shall open a PR to restore it.